lib/hydra/derivatives.rb

```diff
@@ -118,7 +118,7 @@ def constantize_processor(processor)
     end
 
     def transform_datastream(file_name, transform_directives, opts={})
```

Comment:
Well if the change in this pull request is applied then the behaviour will be as before. Passing an empty options hash isn't a problem as long as you transforming an image datastream.